Automattic/newspack-newsletters
# fix: buttons' border radius, width, padding and colors

### All Submissions:






### Changes proposed in this Pull Request:

Fixes button block support for:

 - Border radius
 - Text color
 - Background color
 - Padding
 - Width
 
| Editor | Rendered markup |
| --- | --- |
|  <img width="552" alt="image" src="https://github.com/Automattic/newspack-newsletters/assets/820752/a8cd93d8-7034-4855-bca4-51299f64fb3e"> |  <img width="576" alt="image" src="https://github.com/Automattic/newspack-newsletters/assets/820752/3336d239-9f06-495e-b3e1-3bb1e6ccec62"> |

### How to test the changes in this Pull Request:

1. Draft a new newsletter and add 2 button blocks
2. Style one of the button blocks with custom values for all the style attributes listed above
3. Save and click to preview the newsletter
4. Confirm the preview markup reflects the editor styles for both the default styles and the edited one
5. Send the newsletter and confirm it also renders as expected

### Other information:
